Hardware/HDD/SmartAttribute.cs

```diff
@@ -1,19 +1,14 @@
-﻿/*
- 
-  This Source Code Form is subject to the terms of the Mozilla Public
-  License, v. 2.0. If a copy of the MPL was not distributed with this
-  file, You can obtain one at http://mozilla.org/MPL/2.0/.
- 
-  Copyright (C) 2011-2015 Michael Möller <mmoeller@openhardwaremonitor.org>
-  Copyright (C) 2011 Roland Reinl <roland-reinl@gmx.de>
-	
-*/
+﻿// Mozilla Public License 2.0
+// If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// Copyright (C) LibreHardwareMonitor and Contributors
+// All Rights Reserved
 
 using System;
 using System.Collections.Generic;
+using OpenHardwareMonitor.Interop;
 
 namespace OpenHardwareMonitor.Hardware.HDD {
-    public class SmartAttribute {
+  internal class SmartAttribute {

```

Comment:
I think this can stay public, it might be useful in the future to be able to read smart attributes too.

Reply:
then i have to make Kernel32 also public, 
`public float ConvertValue(Kernel32.DriveAttributeValue value,...`

Reply:
The ConvertValue method can be internal instead, I don't think this method is very useful for public end.

Reply:
Kernel32 can remain internal then as well.